test/unit/ElectronicResources.test.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import { mount } from 'enzyme'
+import { expect } from 'chai'
+
+import ElectronicResources from "../../src/app/components/BibPage/ElectronicResources";
+
+const oneResource = [{ url: "books.com", label: "View on books.com" }]
+const threeResource = [{ url: "books.com/1", label: "View on books.com 1" }, { url: "books.com/2", label: "View on books.com 2" }, { url: "books.com/2", label: "View on books.com 3" }]
+
+describe('ElectronicResources', () => {
```

Comment:
Add one test when the component is rendered without any resources, either `<ElectronicResources />` or `<ElectronicResources electronicResources={[]} />` to cover that case. You may need to update the `ElectronicResources` component itself to handle this. It can be as simple as 
```
if (!electronicResources || !lelectronicResources.length) {
  return null;
}
```